config/server.py

```diff
@@ -244,7 +244,7 @@
     " is only scheduled after results are received in the previous poll.")
 
 config_lib.DEFINE_string(
-    "Server.username", None,
+    "Server.username", 'root',
     "System account to run as after initialization for running the server as "
```

Comment:
"System account to run as after initialization" since the rest doesn't really make sense now.